style.css

```diff
@@ -0,0 +1,828 @@
+}
+
+.feed-tweet-actions {
+    display: grid;
+    grid-template-columns: 1fr 30px;
+    gap: 30px;
+    align-items: center;
+}
+
+.direct-tweet-reaction {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.direct-tweet-reaction > a {
+    display: grid;
+    grid-template-columns: 30px auto;
+    align-items: center;
+}
+
+.direct-tweet-reaction > a > div {
+    display: grid;
+    align-items: center;
+    color: gray;
+    font-size: 12px;
+    width: fit-content;
+}
+
+.direct-tweet-reaction > a > div > img {
+    width: 30px;
+    padding: 4px;
+    border-radius: 30px;
+}
+
+/* REPLY REACTIONS */
+.reply-reactions {
+    text-decoration: none;
+}
+
+.reply-reactions:hover .reply {
+    filter: invert(41%) sepia(46%) saturate(2810%) hue-rotate(195deg) brightness(103%) contrast(102%);
+}
+
+.reply-reactions:hover > div:nth-of-type(1) {
+    border-radius: 30px;
+    background-color: rgba(30, 143, 255, 0.274);
+}
+
+.reply-reactions:hover > .reply-count {
+    color: dodgerblue;
+}
+
+/* REPOST REACTIONS */
+.repost-reaction {
+    text-decoration: none;
+}
+
+.repost-reaction:hover .repost {
+    filter: invert(46%) sepia(89%) saturate(1171%) hue-rotate(88deg) brightness(94%) contrast(101%);
+}
+
+.repost-reaction:hover > div:nth-of-type(1) {
+    border-radius: 30px;
+    background-color: #02a01445;
+}
+
+.repost-reaction:hover .repost-count {
+    color: #03ba18;
+}
+
+/* LIKE REACTIONS */
+.like-reaction {
+    text-decoration: none;
+}
+
+.like-reaction:hover .like {
+    filter: invert(20%) sepia(62%) saturate(7458%) hue-rotate(318deg) brightness(85%) contrast(100%);
+}
+
+.like-reaction:hover > div:nth-of-type(1) {
+    border-radius: 30px;
+    background-color: #d80b7834;
+}
+
+.like-reaction:hover > .like-count {
+    color: #d80b78;
+}
+
+/* VIEW REACTIONS */
+.views-reaction {
+    text-decoration: none;
+}
+
+.views-reaction:hover .views {
+    filter: invert(41%) sepia(46%) saturate(2810%) hue-rotate(195deg) brightness(103%) contrast(102%);
+}
+
+.views-reaction:hover > div:nth-of-type(1) {
+    border-radius: 30px;
+    background-color: rgba(30, 143, 255, 0.274);
+}
+
+.views-reaction:hover > .view-count {
+    color: dodgerblue;
+}
+
+/* SHARE REACTIONS */
+
+.share {
+    display: grid;
+    align-items: center;
+    justify-self: start;
+    width: 22px;
+}
+
+.share-image-bg {
+    width: fit-content;
+    padding: 4px;
+}
+
+.share-reaction:hover .share {
+    filter: invert(41%) sepia(46%) saturate(2810%) hue-rotate(195deg) brightness(103%) contrast(102%);
+}
+
+.share-reaction:hover > div {
+    border-radius: 30px;
+    background-color: rgba(30, 143, 255, 0.274);
+}
+
+/* OTHER ICON ADJUSTMENTS */
+.direct-tweet-reaction > a > div > .reply {
+    width: 28px;
+}
+
+.direct-tweet-reaction > a> div > .views {
+    width: 27px;
+}
+
+/* SPECIFIC POST ADJUSTMENTS */
+
+.post-1 .tweet-photo {
+    border-radius: 20px;
+    object-fit: cover;
+    width: 400px;
+    height: 400px;
+}
```

Comment:
We need some hover effects on these like you've done with `sub-card` on your trending menus! 😇

Reply:
Haha i'll add it to my to-do-list!